tools/terraform_sync_tool/terraform_sync.py

```diff
@@ -0,0 +1,60 @@
+import json
+import io
+from google.cloud import bigquery
+
+# Fetch schemas for drifted tables form BQ
+def get_schemas_from_BQ(drifted_tables):
+    table_schemas = []
+    client = bigquery.Client()
+    for table_id in drifted_tables:
+        table = client.get_table(table_id)
+        file = io.StringIO("")
+        client.schema_to_json(table.schema, file)
+        # append schema as a dict where key=table_id, value=table_schema
+        schema_bq = dict({table_id: json.loads(file.getvalue())}) 
+        table_schemas.append(schema_bq)
+    return table_schemas
+
+# Convert table resource from terraform log output to 
+# table_id format:[gcp_project_id].[dataset_id].[table_id]
+def convert_to_table_id(input):
+    table_id = ""
+    for s in input.rsplit("/"):
+        if(s != "projects" and s != "datasets" and s != "tables"):
+            table_id += s+"."
+    return table_id[:len(table_id)-1]
+
+# Opening JSON file
+f = open('./state.json', 'r')
+data = f.read()
+data = data.split("}\n")
+data = [d.strip() + "}" for d in data]
+data = list(filter(("}").__ne__, data))
+data = [json.loads(d) for d in data]
```

Comment:
The input file is already newline delimited data. Do not use strip and filters, simply load each line using json.loads